data/mods/gen2/moves.ts

```diff
@@ -314,6 +314,9 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			for (const pokemon of source.side.pokemon) {
 				pokemon.clearStatus();
 			}
+			for (const ally of target.side.pokemon) {
```

Comment:
Not necessary to override here, because Gen 2 doesn't have doubles.

Reply:
Good point! I will remove this override then.